### PR TITLE
[WIP] ImportError: cannot import name 'librmm' resolved in Dockerfile

### PR DESCRIPTION
**Pull request by a-agrz**
Even after the changes in the Dockerfile in #404
I encountered the same errors while importing cudf library !!

root@f33fa2ff1be4:/# source activate cudf
(cudf) root@f33fa2ff1be4:/# python -c "import cudf"
Traceback (most recent call last):
File "", line 1, in
File "/conda/envs/cudf/lib/python3.6/site-packages/cudf-0.3.0+138.gdae6026-py3.6-linux-x86_64.egg/cudf/init.py", line 2, in
from cudf import dataframe # noqa: F401
File "/conda/envs/cudf/lib/python3.6/site-packages/cudf-0.3.0+138.gdae6026-py3.6-linux-x86_64.egg/cudf/dataframe/init.py", line 1, in
from cudf.dataframe import (buffer, dataframe, series, # noqa: F401
File "/conda/envs/cudf/lib/python3.6/site-packages/cudf-0.3.0+138.gdae6026-py3.6-linux-x86_64.egg/cudf/dataframe/buffer.py", line 3, in
from librmm_cffi import librmm as rmm
ImportError: cannot import name 'librmm'
(cudf) root@f33fa2ff1be4:/#

A turn around was to install cudf directly using 'conda install' as described in the README.md file
By doing this I can import cudf in the docker container without a problem 
Hop this helped  
Best regards

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

File: Dockerfile [hunk 1]
```diff
@@ -61,6 +63,8 @@ RUN source activate cudf && \
     make install_python
 
 # cuDF build/install
-RUN source activate cudf && \
-    cd /cudf/python && \
-    python setup.py install
+#RUN source activate cudf && \
+#    cd /cudf/python && \
+#    python setup.py install
+RUN conda install -c nvidia -c rapidsai -c numba -c conda-forge -c defaults cudf=0.3.0
```
Comment (kkraus14) [inline Dockerfile]: We shouldn't install from conda in the container as this then ignores the branch specified by an arg earlier and prevents using the container for development.

File: Dockerfile [hunk 2]
```diff
@@ -30,12 +30,14 @@ RUN conda create -n cudf python=${PYTHON_VERSION}
 
 ARG NUMBA_VERSION=0.40.0
 ARG NUMPY_VERSION=1.14.3
+ARG CYYHON_VERSION=0.29.1
```
Comment (kkraus14) [inline Dockerfile]:
```suggestion
ARG CYTHON_VERSION=0.29.1 
```

File: Dockerfile [hunk 3]
```diff
@@ -30,12 +30,14 @@ RUN conda create -n cudf python=${PYTHON_VERSION}
...
 ARG NUMBA_VERSION=0.40.0
 ARG NUMPY_VERSION=1.14.3
+ARG CYYHON_VERSION=0.29.1 
 # Locked to Pandas 0.20.3 by https://github.com/rapidsai/cudf/issues/118
 ARG PANDAS_VERSION=0.20.3
 ARG PYARROW_VERSION=0.10.0
 RUN conda install -n cudf -y -c numba -c conda-forge -c nvidia -c rapidsai -c defaults \
       numba=${NUMBA_VERSION} \
       numpy=${NUMPY_VERSION} \
+      cython=${CYYHON_VERSION}\
```
Comment (kkraus14) [inline Dockerfile]:
```suggestion
      cython=${CYTHON_VERSION}\
```

**Comment by harrism** [comment]
Is this a valid pull request now that 0.4.0 is released? I think a similar issue was solved by adding cffi to the conda install in the Docker container?

**Comment by kkraus14** [comment]
Closing as this is resolved in 0.4 release. Please open a new PR if further changes are needed.

**Pull request closed without merging**